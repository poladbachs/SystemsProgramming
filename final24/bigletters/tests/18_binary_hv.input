0100010110111010
