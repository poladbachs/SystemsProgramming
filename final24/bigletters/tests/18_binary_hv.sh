#!/bin/bash
exec "$1" tests/BIT_FONT3 < tests/18_binary_hv.input

