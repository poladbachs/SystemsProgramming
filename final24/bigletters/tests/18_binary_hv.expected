      1                   1       1 1       1 1 1       1
      1                   1       1 1       1 1 1       1
00000 1 00000 00000 00000 1 00000 1 1 00000 1 1 1 00000 1 00000
      1                   1       1 1       1 1 1       1
      1                   1       1 1       1 1 1       1
